samples/nrf9160/gps/src/supl_support.h

```diff
@@ -0,0 +1,69 @@
+
+/** This function will open an socket to the SUPL server
+ *
+ * @brief This function will create and open a socket to the SUPL servers
+ *         that can be used by the SUPL clients read and write functions.
+ *         The socket handle is available in the implementation .c file scope
+ *         and a reference is therefor not passed to the function.
+ *
+ * @retval 0  If opening socket was successful.
+ *         -1 If opening socket failed
+ */
+int     open_supl_socket(void);
```

Comment:
Excess spaces here and in the declarations below.

Reply:
hmmm, strange, fixed now.